src/main/flight/pid.h

```diff
@@ -76,6 +76,12 @@ typedef struct pid8_s {
     uint8_t D;
 } pid8_t;
 
+typedef enum {
+    ANTI_GRAVITY_OFF = 0,
```

Comment:
To turn off antigravity we have a feature or switch box, why have third place where it can be turned on of off, its just mode so leave the option to handle only STEP or SMOOTH mode

Reply:
OK that makes sense